guppylang/prelude/_internal.py

```diff
@@ -67,6 +74,11 @@ def float_value(f: float) -> val.Value:
     return val.ExtensionVal(c=(ConstF64(value=f),))
 
 
+def list_value(v: list[val.Value]) -> val.Value:
+    """Returns the Hugr representation of a list value."""
+    return val.ExtensionVal(c=(ListValue(value=v),))
```

Comment:
What does the `,` do here?

Reply:
`(x,)` is a unary tuple. We need this because of some serialisation shenanigans...